app/Http/Controllers/Smartcars/Api/Authentication.php

```diff
@@ -37,7 +37,7 @@ public function postManual()
             return 'AUTH_FAILED';
         }
 
-        if ($account->is_banned) {
+        if ($account && $account->is_banned) {
```

Comment:
This is simply cascading the error further down the code, so it will now occur on line 44 instead, when `->verifyPassword()` is called on the same object.

We should simply return AUTH_FAILED when the account is first retrieved if it's null - then we can remove any further `$account` checks in the function, which will mean the same error isn't made in future when someone writes another line which depends upon the `$account` object.